en/clubs/informationsorganet/body.md

```diff
@@ -35,4 +35,4 @@ Here is a list of some systems that previous students have built and that the ch
 
 ## Join the club
 
-We regularly try to hold programming evenings where everyone is welcome. Usually from 5.30 pm onwards on Thursdays, sometimes in a computer room and sometimes on Zoom. You can come here to see what we're doing, develop something for the chapter, hang out or maybe get help with a programming lab. The easiest way to find out when we have events is to join our [slack](https://ior.slack.com). Hope to see you there!!
+We regularly try to hold programming evenings where everyone is welcome. Usually from 5 pm onwards on Thursdays, usually somewhere in the E-house. You can come here to see what we're doing, develop something for the chapter, hang out or maybe get help with a programming lab. The easiest way to find out when we have events is to ask in our [slack](https://ior.slack.com) or Head of IT at [d-sys@datasektionen.se](mailto:d-sys@datasektionen.se). Hope to see you there!
```

Comment:
The problem with discord is that you need to be invited to see the ior channel, while the slack is open for everyone to join, and even though it's not really used any more it's still possible to ask questions there :p
On the other hand, you can just as well ask for the time/location in the general chat channel on discord. I also think that we should use some open channel that any chapter member can join instead of the private discord channel, but I'm not a fan of slack (is anyone?)